File: api/src/main/java/club/yunzhi/api/workReview/entity/Site.java
package club.yunzhi.api.workReview.entity;

import javax.persistence.Entity;

/**
 * 井场实体
 */
@Entity
public class Site extends BaseEntity {
    /**
     * 井场名称
     */
    private String name;

    /**
     * 井场编号
     */
    private String code;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }
}

File: ant-design-vue-pro-master/src/entity/Site.js
export default class Site {
  constructor({
    id = '',
    name = '',
    code = '',
    createTime = new Date(),
    updateTime = new Date()
  } = {}) {
    this.id = id
    this.name = name
    this.code = code
    this.createTime = createTime
    this.updateTime = updateTime
  }
}
